packages/jsii-pacmak/lib/targets/go.ts

```diff
@@ -53,23 +54,36 @@ class GoGenerator implements IGenerator {
     });
   }
 
-  public async save(outDir: string, _tarball: string): Promise<any> {
+  public async save(outDir: string, tarball: string): Promise<any> {
     // TODO: Replace with [go-embed](https://github.com/pyros2097/go-embed)
-    // const bundledRuntimeDotGo = path.join('_jsii', 'bundled-runtime.go');
-    // this.code.openFile(bundledRuntimeDotGo);
+    const bundledRuntimeDotGo = path.join('_jsii', 'bundled-runtime.go');
+    this.code.openFile(bundledRuntimeDotGo);
```

Comment:
This file was opened, but never closed... Perhaps this explains why it does not show in the output?